tests/test_wrappers.py

```diff
@@ -0,0 +1,26 @@
+from benchmark_environments.testing.envs import CountingEnv
+from benchmark_environments.util import AutoResetWrapper
+
+
+def test_auto_reset_wrapper(episode_length=3, n_steps=100, n_manual_reset=2):
+    env = AutoResetWrapper(CountingEnv(episode_length=episode_length))
+
+    for _ in range(n_manual_reset):
+        obs = env.reset()
+        assert obs == 0
+
+        for t in range(1, n_steps + 1):
+            act = env.action_space.sample()
+            obs, rew, done, info = env.step(act)
+            expected_obs = t % episode_length
+
+            assert obs == expected_obs
+            assert done is False
+
+            # End of episode
+            if expected_obs == 0:
```

Comment:
Consider moving comment down like:
`if expected_obs == 0:  # end of episode`

Where it currently is I initially interpreted it as applying to the entire `if-else` block.